pyabc/visualization/histogram.py

```diff
@@ -60,6 +62,8 @@ def plot_histogram_1d_lowlevel(
         range_ = (xmin, xmax)
     else:
         range_ = None
+    if refval is not None:
+        ax.axvline(refval[x], color='C1', linestyle='dashed')
```

Comment:
possibility to change refval color? why not in black?

Reply:
not black, because we also use it on the (black) scatter plots. But I agree that 'C1' is not ideal. Until now, it always worked, so I would leave it until we encounter an application problem.